tac/project-updates/2025/2025-Q2-Web3j.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: 2025 Q2 Web3j
+parent: 2025
+grand_parent: Project Updates
+---
+
+# Project Health
+
+The project is still relatively new to the LF Decentralized Trust Ecosystem. It was migrated to Hyperledger Foundation from Web3 Labs organization at the beginning of 2024.
+Contributors are joining the Web3j Contributors happens every 2 weeks, [calendar](https://zoom-lfx.platform.linuxfoundation.org/meetings/lf-decentralized-trust?view=month).
+There is also Discord activity.
+
+# Questions/Issues for the TAC
+
+None
+
+# Releases
+
+LTS release:
+- v4.14.0 is the current LTS release.
+
+[Releases](https://github.com/LFDT-web3j/web3j/releases) past quarter:
+- Fabric v4.14.0 - April 9, 2025
+- Fabric v4.13.0 - March 5, 2025
+- Fabric v4.12.3 - January 3, 2025
```

Comment:
Is this section correct? Specifically, the use of the word Fabric is throwing me off.